packages/devextreme/js/core/utils/throttle.js

```diff
@@ -0,0 +1,13 @@
+export function throttle(func, delay) {
```

Comment:
Because you added a utility in the core area, you need to notify the Essentials team about it and ask them to review it. As far as I know, they are working on a PR for migrating to TS, and your utility could introduce additional difficulties.
https://github.com/DevExpress/DevExtreme/pull/28135 They should take new file into account after merge it

Reply:
We decided to drop the throttle functionality [30f954d8b2d174191c45c26c9ef35cce6e9da515]